src/main/java/org/purpurmc/purpurextras/modules/CreateSusBlocksModule.java

```diff
@@ -32,7 +40,16 @@ public void enable() {
 
     @Override
     public boolean shouldEnable() {
-        return PurpurExtras.getPurpurConfig().getBoolean("settings.create-suspicious-blocks", false);
+        boolean susBlocksEnabled = PurpurExtras.getPurpurConfig().getBoolean("settings.suspicious-blocks.enabled", PurpurExtras.getPurpurConfig().getBooleanIfExists("settings.create-suspicious-blocks", false));
+        this.exclusionListStatus = PurpurExtras.getPurpurConfig().getBoolean("settings.suspicious-blocks.enable-item-exclusion-list", false);
```

Comment:
I don't think this toggle is necessary here, it's as good as making the list empty (`[]`)

Reply:
i’m personally more in favour of the toggle here so that the users can specify the list and if they need to disable it they don’t have to delete its contents but just flip the toggle